pkg/management/authorizationutil.go

```diff
@@ -53,3 +54,42 @@ func checkTarget(authorizations []configuration.Authorization, allowedTargetReal
 	}
 	return nil
 }
+
+// Validate the scope of each authorization in the array. Returns an error if an authorization is not valid
+func ValidateScope(authorizations []configuration.Authorization) error {
+	for _, authz := range authorizations {
+		scope, err := getScope(authz)
+		if err != nil {
+			return err
+		}
+
+		if authz.TargetRealmID == nil {
+			return errors.New("missing target realm")
+		}
+
+		switch scope {
+		case security.ScopeGlobal:
+			if *authz.TargetRealmID != "*" || authz.TargetGroupName != nil {
+				return errors.New("invalid global scope")
```

Comment:
`return errorhandler.CreateBadRequestError(constants.MsgErrInvalidParam + "." + constants.Authorization+".scope")` ?

Reply:
I did that to have the same kind of error of the method Validate from the same file, I then return `errorhandler.CreateBadRequestError(constants.MsgErrInvalidParam + "." + constants.Authorization)` when I used it. Example : https://github.com/cloudtrust/keycloak-bridge/blob/4961ed1d33150cddad65dbaaa7db1b6eddc37eff/pkg/management/component.go#L1475